scripts/fastsp_script/fsp_test.py

```diff
@@ -0,0 +1,41 @@
+import os
+import sys
+
+"""
+###remember sys.argv - can create variables using indexes with sys.argv!  
+    
+"""
+
+dir = "perturbed_alignments_Drosophila_AA/EMGT00050000000018.Drosophila.001_AA"
+slash = "/"
+
+#print(sys.argv)
+
+outfile = output_fs.txt
+#will need to be .csv later 
+
+aa_ls_dir = os.listdir(dir)
+
+for file in aa_ls_dir:
```

Comment:
The loop is definitely doubling comparisons - do something like this instead:
```
for i in range(5):
    for j in range(i,5):
        if i !=j:
            print(i,j)
```
can reference file names as `aa_ls_dir[i]` and `aa_ls_dir[j]`

Reply:
Did it!